specification/devcenter/DevCenter/DevBox/models.tsp

```diff
@@ -454,21 +599,102 @@ model DevBoxNextAction {
   scheduledTime: utcDateTime;
 }
 
-@doc("The actions list result")
+@doc("The actions list result.")
 model DevBoxActionsDelayMultipleResult
   is Azure.Core.Page<DevBoxActionDelayResult>;
 
-@doc("The action delay result")
+@doc("The action delay result.")
 model DevBoxActionDelayResult {
+  @doc("The unique URI of the action.")
+  @added(APIVersions.v2024_02_01)
+  uri: url;
+
   @doc("The name of the action.")
   name: string;
 
   @doc("The result of the delay operation on this action.")
   result: DevBoxActionDelayResultStatus;
 
-  @doc("The delayed action")
+  @doc("The delayed action.")
   action?: DevBoxAction;
 
   @doc("Information about the error that occurred. Only populated on error.")
   error?: Azure.Core.Foundations.Error;
 }
+
+@doc("Information about an operation on a Dev Box.")
+@resource("operations")
+@parentResource(DevBox)
+@added(APIVersions.v2024_02_01)
+@discriminator("kind")
+model DevBoxOperation {
+  @doc("The unique URI for the Dev Box operation.")
+  uri: url;
+
+  @doc("Unique identifier for the Dev Box operation.")
+  @visibility("read")
+  @key("operationId")
+  operationId: Azure.Core.uuid;
+
+  @doc("The kind of operation that occurred.")
+  kind: DevBoxOperationKind;
+
+  @doc("The operation status.")
+  status: DevBoxOperationStatus;
```

Comment:
Can we use the operation status type provide by tsp azure core? https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/data-types#Azure.Core.Foundations.OperationStatus

Reply:
Then we can remove DevBoxOperationStatus

Reply:
Changed to `Azure.Core.Foundations.OperationState`